src/Catty/FormulaEditor/ViewController/FormulaEditorViewController.m

```diff
@@ -710,16 +700,6 @@ - (void)showNotification:(NSString*)text andDuration:(CGFloat)duration
                                         inView:self.view completion:^{ [self.notficicationHud removeFromSuperview]; }];
 }
 
-- (void)showChangesSavedView
-{
-    [self showNotification:kUIFEChangesSaved andDuration:kBDKNotifyHUDPresentationDuration];
```

Comment:
Please check if we need the deprecated `showNotification` method. If not, please remove it.